monitors/health_check.py

```diff
@@ -70,12 +72,44 @@ def get_db_client():
         Login to the database
         :return: Database client
         """
-        db_cli = DatabaseClient()
+        db_client = DatabaseClient()
+        try:
+            db_client.connect()
+        except ConnectionException:
+            logging.error("Unable to connect to Database")
+        return db_client
+
+    @staticmethod
+    def resubmit_run(icat_client, instrument, run_number):
+        """
+        Resubmit runs that are missing (have not been submitted by end of run monitor)
+        :param instrument: The instrument associated with the missing run
+        :param run_number: The run number to resubmit
+        :param limit: Where we expect the rb number to be located in the file
+        :param use_nxs: The expected extension of the data file
+        """
+        # Connect to the autoreduce queues
+        queue_client = QueueClient()
         try:
-            db_cli.connect()
+            queue_client.connect()
         except ConnectionException:
-            logging.error("Unable to connect to MySQL")
-        return db_cli
+            logging.error("Unable to connect to Queue")
+            return False
```

Comment:
It might be worth opening a separate opening a separate issue to add functionality in the clients to perform a none throwing connect. I can imagine this is a fairly common issue that we'll potentially encounter more than just here.